### ISSUE TITLE
Make sure plone.org frontpage does not have contrast issues (accessibility) 

### ISSUE THREAD

**Issue by rioksane** [plone/ploneorg.core#302]
Refering to a longer disucssion https://github.com/plone/ploneorg.core/issues/147

If checking with Wave toolbar, plone.org home page has 2 contrast errors: 
![Screenshot 2021-03-21 at 18 02 07](https://user-images.githubusercontent.com/5127672/111911791-97efa500-8a6f-11eb-9424-da243774f5d2.png)

1. the greenish button on hero section - fix with css
2. Open chat - fix with integration settings(?)
